rstudio.tera

```diff
@@ -0,0 +1,812 @@
+---
+whiskers:
+  version: "2.2.0"
+  matrix:
+    - flavor
+  filename: "catppuccin-{{ flavor.identifier }}.rstheme"
+---
+
+{%- set surface1_80 = surface1 | mod(opacity=0.8) -%}
+{%- set surface1_32 = surface1 | mod(opacity=0.32) -%}
+
+/* rs-theme-name: catppuccin-{{ flavor.identifier }} */
+/* rs-theme-is-dark: {% if flavor.dark %}TRUE{% else %}FALSE{% endif %} */
+
+:root {
+{%- for _, color in flavor.colors %}
+  --{{ color.identifier }}: #{{ color.hex }};
+{%- endfor %}
+}
+
+.ace_gutter {
+  background: var(--base);
+  color: var(--subtext0); /* other theme sets this to #838995, but this is not part of Nord palette */
```

Comment:
Is the comment here still relevant? It'd be nice to clarify what `other theme` is and why that matters (and why we mention the Nord palette)

Reply:
Not sure, just going with the small diff policy. That would be a question for the (previous?) maintainer of this port.

Reply:
Hey :wave: @AngelBerihuete,

Would you be able to provide us with some context on what `other theme` here is? And why the Nord palette has been mentioned?